lib/Sema/TypeCheckDecl.cpp

```diff
@@ -1910,8 +1886,7 @@ FunctionOperatorRequest::evaluate(Evaluator &evaluator, FuncDecl *FD) const {
       opDiagnostic.fixItInsert(insertionLoc, insertion);
     return nullptr;
   }
-
-  return op;
+  return op.get();
```

Comment:
It's not evident in the diff, but we check this pointer right?

Reply:
Yup, there's an `if (!op)` statement just above this that emits a diagnostic and bails.

Thanks for the review!